src/components/App/Select/index.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { fetchEpisodes } from 'redux/actions/episodesActions';
+import PropTypes from 'prop-types';
+
+import styles from './Select.module.scss';
+
+class Select extends React.Component {
+  render() {
+    return (
+      <select onChange={e => this.props.onChange(e.target.value)} value={this.props.value} className={styles.customSelect}>
+        {this.props.options.map(option => (
+          <option value={option.code} key={option.code}>
+            {option.name}
+          </option>
+        ))}
+      </select>
+    );
+  }
+}
+
+const mapDispatchToProps = dispatch => (
+  bindActionCreators({ fetchEpisodes }, dispatch)
+);
+
+const mapStateToProps = state => ({
+  episodes: state.data.episodes,
+});
+
+Select.propTypes = {
+  options: PropTypes.arrayOf(PropTypes.shape({
```

Comment:
Do you have Prettier enabled? Something is wrong with indents

Reply:
It looks fine on my machine, however it's true that I had wrong indents set on my new Mac so I'l;l double check it. :)